Runtime/Autopilot.cs

```diff
@@ -8,9 +8,11 @@
 using DeNA.Anjin.Loggers;
 using DeNA.Anjin.Settings;
 using DeNA.Anjin.Utilities;
+using UnityEditor;
```

Comment:
[ASK]
Maybe `#if UNITY_EDITOR ... #endif` symbol guard is necessary?

Reply:
Thanks! fix in 95acd366